x-pack/docs/en/ilm/apis/set-policy.asciidoc

```diff
@@ -0,0 +1,96 @@
+[role="xpack"]
+[testenv="basic"]
+[[ilm-put-policy]]
+=== Set Policy On Index API
+++++
+<titleabbrev>Set Policy On Index</titleabbrev>
+++++
+
+Assigns a policy to an index for management.
+
+==== Request
+
+`POST <index>/_ilm/<policy>`
+
+==== Description
+
+This action assigns a policy to an index. It is effectively the same as setting an index's
```

Comment:
This should be updated to mention that `<index>` can be a comma separated list of indices, and (I think) can contain wildcards